readthedocs/core/static-src/core/js/doc-embed/rtd-data.js

```diff
@@ -17,9 +22,9 @@ var configMethods = {
     get_theme_name: function () {
         // Crappy heuristic, but people change the theme name on us.  So we have to
         // do some duck typing.
-        if (this.theme !== 'sphinx_rtd_theme') {
+        if (this.theme !== constants.THEME_RTD) {
             if ($('div.rst-other-versions').length === 1) {
```

Comment:
Should we also have heuristics around alabaster, too? Seems like we'd want to handle folks renaming it as well.